app/services/audit_results.rb

```diff
@@ -14,12 +14,12 @@ class AuditResults
 
   INVALID_ARGUMENTS = :invalid_arguments
   VERSION_MATCHES = :version_matches
-  ARG_VERSION_GREATER_THAN_DB_OBJECT = :arg_version_greater_than_db
-  ARG_VERSION_LESS_THAN_DB_OBJECT = :arg_version_less_than_db
+  ACTUAL_VERS_GT_DB_OBJ = :arg_version_greater_than_db
+  ACTUAL_VERS_LT_DB_OBJ = :arg_version_less_than_db
   CREATED_NEW_OBJECT = :created_new_object
   DB_UPDATE_FAILED = :db_update_failed
-  OBJECT_ALREADY_EXISTS = :object_already_exists
-  OBJECT_DOES_NOT_EXIST = :object_does_not_exist
+  DB_OBJ_ALREADY_EXISTS = :object_already_exists
+  DB_OBJ_DOES_NOT_EXIST = :object_does_not_exist
   PC_STATUS_CHANGED = :pc_status_changed
```

Comment:
Similarly, `OBJECT` changes to `DB_OBJ` on the left, but `object` stays `object` on the right?

Reply:
also ... oops.  Will fix.